perl/bin/cgpVaf.pl

```diff
@@ -68,25 +67,25 @@ BEGIN
     my $progress_hash;
     # progress checked before the processing starts , speed ups concatenation step 
     my ($chromosomes)=$vcf_obj->getChromosomes($options->{'chr'});
-    ($progress_hash,$chromosomes)=$vcf_obj->getProgress($chromosomes);
-    
+    $chromosomes=$vcf_obj->getProgress($chromosomes);
+
     # this is called only once to add allSample names to vcf object
     $vcf_obj->getAllSampleNames;
     my($info_tag_val,$vcf_file_obj)=$vcf_obj->getVcfHeaderData;
-    my($bam_objects,$bas_files)=$vcf_obj->_get_bam_object;
-    my($bam_header_data,$lib_size)=$vcf_obj->_get_bam_header_data($bam_objects,$bas_files);
+    my($bam_objects,$bas_files)=$vcf_obj->get_bam_object;
+    my $max_lib_size = $vcf_obj->get_lib_n_read_read_length($bam_objects,$bas_files);
     # create variant object
     my $variant=Sanger::CGP::Vaf::Process::Variant->new(
         'location'      => undef,
         'tmp'           =>$options->{'tmp'},
         'varLine'       => undef,
         'varType'       => $vcf_obj->{'_a'},
-        'libSize'       => defined $lib_size?$lib_size:100,
         'samples'       => $vcf_obj->{'allSamples'},
         'tumourName'    => $vcf_obj->getTumourName,
         'normalName'    => $vcf_obj->getNormalName,
         'vcfStatus'     => $vcf_obj->{'vcf'},
         'noVcf'         => defined $vcf_obj->{'noVcf'}?$vcf_obj->{'noVcf'}:undef,
+        'lib_size'      => $max_lib_size,
```

Comment:
Does this change default behaviour in any way?  Additions to command line args are acceptable in hotfixes only if no change to default behaviour

Reply:
Ideally shouldn't as I am choosing max library size for an individual sample group.
This will avoid issues due to smaller library size in one of the samples in a group.
Number of reads aligned at variant/normal position should be same.